### PR TITLE
[MODULE] Migrate tests reconst noddi and diffusivitypriors

### PR DESCRIPTION
**Pull request by arnaudbore**
## Type of improvement

**If submitting a new module or fixing a bug, please use the appropriate template.**

- [ ] Documentation
- [ ] Development tools (e.g. linter, formatter, etc.)
- [ ] Development container
- [x] Global update (please specify)
- [ ] Other (please specify)

## Describe your improvement

Migrate module reconst/ noddi and diffusivitypriors to scilpy 2.0.2 and nf-test framework.

## Checklist before requesting a review

- [x] Ensure the syntax is correct (**EditorConfig** and **Prettier** must pass)
- [x] Run the test suites if your changes affect any module
- [ ] Regenerate the **Poetry** lock file if you have updated the dependencies
- [ ] Ensure the documentation is up-to-date

### CODE REVIEW

**Comment by AlexVCaron** [comment]
Closes #79

**Comment by arnaudbore** [comment]
@AlexVCaron @gagnonanthony @ThoumyreStanislas I would need your help on this one. :+1:

File: modules/nf-scil/reconst/diffusivitypriors/meta.yml [hunk 1]
```diff
@@ -28,6 +28,11 @@ input:
       description: Nifti DTI AD volume.
       pattern: "*.{nii,nii.gz}"
 
+  - rd:
+      type: file
+      description: Nifti DTI AD volume.
```
Comment (gagnonanthony) [inline modules/nf-scil/reconst/diffusivitypriors/meta.yml:33]:
```suggestion
      description: Nifti DTI RD volume.
```

File: modules/nf-scil/reconst/diffusivitypriors/meta.yml [hunk 2]
```diff
@@ -50,6 +55,11 @@ output:
       description: Text files for para diffusivity values.
       pattern: "*.txt"
 
+  - perp_diff:
+      type: file
+      description: Text files for para diffusivity values.
```
Comment (gagnonanthony) [inline modules/nf-scil/reconst/diffusivitypriors/meta.yml:60]:
```suggestion
      description: Text files for perpendicular diffusivity values.
```

File: modules/nf-scil/reconst/diffusivitypriors/meta.yml [hunk 3]
```diff
@@ -60,14 +70,19 @@ output:
...
+
   - mean_iso_diff:
       type: file
       description: Text file for average iso diffusivity value.
       pattern: "*.txt"
 
   - priors:
       type: directory
-      description: Folder containg text files for iso and para diffusivity values.
+      description: Folder containing text files for iso and para diffusivity values.
```
Comment (gagnonanthony) [inline modules/nf-scil/reconst/diffusivitypriors/meta.yml:85]:
```suggestion
      description: Folder containing text files for iso, para, and perp diffusivity values.
```

**Review by gagnonanthony — APPROVED** [review]
Minor changes in the meta.yml, but otherwise GTG 🍻

File: modules/nf-scil/reconst/noddi/tests/main.nf.test [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+                ]}
+            """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.versions,
+                    file(process.out.kernels.get(0)).listFiles().size()
```
[the remark this answers is not in the document]
Comment (arnaudbore) [reply modules/nf-scil/reconst/noddi/tests/main.nf.test:78]: Not in this example.

**Pull request merged**